src/core/lib/channel/channel_args.cc

```diff
@@ -257,6 +257,14 @@ int grpc_channel_arg_get_integer(const grpc_arg* arg,
   return arg->value.integer;
 }
 
+int grpc_channel_args_find_integer(const grpc_channel_args* args,
+                                   const char* name,
+                                   const grpc_integer_options options) {
+  const grpc_arg* arg = grpc_channel_args_find(args, name);
+  if (arg == nullptr) return options.default_value;
```

Comment:
No need for this check, since it's done inside of `grpc_channel_arg_get_integer()`.